agilex_piper/piper.xml

```diff
@@ -0,0 +1,300 @@
+<mujoco model="piper_description">
+  <compiler angle="radian" meshdir="assets/" texturedir="assets/" />
+
+  <default>
+    <default class="visual">
+      <geom group="2" type="mesh" contype="0" conaffinity="0" density="0" />
+    </default>
+    <default class="collision">
+      <geom group="3" type="mesh" />
+    </default>
+    <joint damping="0.1" stiffness="1" frictionloss="0.1" armature="0.1" />
+    <general gaintype="fixed" biastype="affine" gainprm="2000" biasprm="0 -2000 -200" />
+  </default>
+
+
+  <asset>
+    <material name="gray_mat" specular="0.0" shininess="0.25" rgba="0.58823 0.58823 0.58823 1" />
+    <material name="light_gray_mat" specular="0.0" shininess="0.25" rgba="0.98039 0.98039 0.98039 1" />
+    <material name="light_medium_gray_mat" specular="0.0" shininess="0.25" rgba="0.85 0.85 0.85 1" />
+    <material name="dark_gray_mat" specular="0.0" shininess="0.25" rgba="0.08628 0.08628 0.08628 1" />
+    <material name="darker_gray_mat" specular="0.0" shininess="0.25" rgba="0.14118 0.14118 0.14118 1" />
+    <material name="white_mat" specular="0.0" shininess="0.25" rgba="1 1 1 1" />
+    <material name="red_mat" specular="0.0" shininess="0.25" rgba="0.81569 0.14902 0.14902 1" />
+    <material name="black_mat" specular="0.0" shininess="0.25" rgba="0 0 0 1" />
+
```

Comment:
1. Can you remove specula and shininess, it doesn't really make a difference.
2. Nit but can you just use 2 significant digits for the rgba values?
3. Only 1 newline between the materials and the meshes.